xml/dhcp_vendor_class.xml

```diff
@@ -203,4 +212,80 @@
     <param pos="0" name="os.family" value="Windows"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+  <fingerprint pattern="^(?i)Cisco Systems Inc. Wireless Phone (\d{4})$">
+    <description>Cisco Wireless Phone</description>
+    <example hw.model="7925">Cisco Systems Inc. Wireless Phone 7925</example>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.certainty" value="0.8"/>
+    <param pos="0" name="os.vendor" value="Cisco"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(?i)Cisco Systems, Inc. IP Phone (.*)">
+    <description>Cisco IP Phone</description>
+    <example hw.model="CP-7911G">Cisco Systems, Inc. IP Phone CP-7911G</example>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.certainty" value="0.8"/>
+    <param pos="0" name="os.vendor" value="Cisco"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Cisco AP (c(?:1130|1140|1240|1250|1260|1600|2600|3500|3600))$" certainty="0.9">
+    <description>Cisco Aironet Wireless Access Point (obsolete)</description>
```

Comment:
Should we combine this pattern with the "Cisco Aironet Wireless Access Point" pattern below? It is the same product so I'm not sure them simply being obsolete warrants a separate fingerprint. At some point in time the models in the other fingerprint will also become obsolete.

Reply:
I believe some customers would find it useful to have obsolete WAPs pointed out since they represent a major security risk, hence the separation. But if the question is rhetorical, I'll combine the two categories.

Reply:
In my experience, few consumers of Recog are actually using the `description` field in a user facing way.  When at R7 I primarily used it for metrics generation and as commentary describing how this fingerprint differs from other similar fingerprints (for example, `w/o version string`, `with hostname`, etc. 

Additionally, the list of "obsolete" products changes over time. IMO, that logic should be left to the consuming application as opposed to pushing the maintenance burden to Recog.

Reply:
Removed the obsolete-Aironet fingerprint (merged into Aironet)